clang/test/CodeGenHLSL/builtins/select.hlsl

```diff
@@ -0,0 +1,76 @@
+// RUN: %clang_cc1 -finclude-default-header -x hlsl -triple \
+// RUN:   dxil-pc-shadermodel6.3-library %s -emit-llvm -disable-llvm-passes \
+// RUN:   -o - | FileCheck %s --check-prefixes=CHECK
+
+// CHECK: %hlsl.select = select i1
+// CHECK: ret i32 %hlsl.select
+int test_select_bool_int(bool cond0, int tVal, int fVal) { return select<int>(cond0, tVal, fVal); }
```

Comment:
nit: https://llvm.org/docs/CodingStandards.html#source-code-width

these would be easier to read in the github PR UI if it were formatted to 80 columns.

Reply:
I agree with @damyanp, that these are easier to read if formatted.

That said, clang doesn't require formatting test cases since formatting sometimes formatting matters to tests. That obviously isn't the case here, but formatting for readability is good.